docs/tutorials/semantic_search/conversation_search_with_byte_quantized_vector.md

```diff
@@ -0,0 +1,278 @@
+# Topic
+
+> This tutorial doesn't explain details about byte-quantized vectors. Read this [Byte-quantized vectors in OpenSearch](https://opensearch.org/blog/byte-quantized-vectors-in-opensearch/) to learn more details.
+
+This tutorial explains how to build semantic search with [Cohere Embedding model](https://docs.cohere.com/reference/embed) and byte-quantized vectors in OpenSearch.
+
+Cohere embedding model v3 supports more `embedding_types`, this tutorial uses `"int8"` to work with byte-quantized vectors.
+
+Note: Replace the placeholders that start with `your_` with your own values.
+
+# Steps
+
+Cohere embedding model v3 supports different `input_type`. From Cohere [doc](https://docs.cohere.com/reference/embed)
+> - `"search_document"`: Used for embeddings stored in a vector database for search use-cases.
+> - `"search_query"`: Used for embeddings of search queries run against a vector DB to find relevant documents.
```

Comment:
IMHO, these descriptions from Cohere are a little hard to understand. Can it be something like:
- `search_document`: Specifies the document vector embeddings stored in a vector database that can be searched.
- `search_query`: Specifies the embeddings of search queries you can use to search documents in a vector DB.

Reply:
@tianjing-li , can you help review ?

Reply:
We also have the following docs if they help clarify their use cases: ```input_type="search_document": Use this when you have texts (documents) that you want to store in a vector database.
input_type="search_query": Use this when structuring search queries to find the most relevant documents in your vector database.

Reply:
Thanks @tianjing-li , can you share the doc link ?

Reply:
https://docs.cohere.com/docs/embed-api#the-input_type-parameter here you go